cmd/commands/runtime.go

```diff
@@ -1424,13 +1447,14 @@ func RunRuntimeList(ctx context.Context) error {
 	}
 
 	tb := ansiterm.NewTabWriter(os.Stdout, 0, 0, 4, ' ', 0)
-	_, err = fmt.Fprintln(tb, "NAME\tNAMESPACE\tCLUSTER\tVERSION\tSYNC_STATUS\tHEALTH_STATUS\tHEALTH_MESSAGE\tINSTALLATION_STATUS\tINGRESS_HOST\tINGRESS_CLASS")
+	_, err = fmt.Fprintln(tb, "NAME\tTYPE\tNAMESPACE\tCLUSTER\tVERSION\tSYNC_STATUS\tHEALTH_STATUS\tHEALTH_MESSAGE\tINSTALLATION_STATUS\tINGRESS_HOST\tINGRESS_CLASS")
```

Comment:
I don't think we need another column for that, we already have enough columns. I think you should do it like the runtime select-i.e. have `(hosted)`. Also people that don't even have the managed runtime feature flag and can't create managed runtimes will started to see this `Hybrid` type here and have question about it.